inc/class-qm-collector-notifications.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Data collector class
+ */
+
+namespace Altis\Workflow;
+
+use QM_Collector;
+
+class QM_Collector_Notifications extends QM_Collector {
+
+	public $id = 'workflow_notifications';
+
+	/**
+	 * Defines the name of the collector.
+	 *
+	 * @return string|void
+	 */
+	public function name() {
+		return __( 'Workflow Notifications', 'query-monitor' );
```

Comment:
Let's change the text domain to `altis`, we've been using that in all the modules.